pages/class/[classid].tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react'
+import type { NextPage } from 'next'
+import { useRouter } from 'next/router'
+import Container from '@mui/material/Container'
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import Grid from '@mui/material/Grid'
+import CircularProgress from '@mui/material/CircularProgress'
+import CommentCard from '../../src/components/CommentCard'
+
+interface Review {
+	semester_id: string
+	rating: number
+	difficulty: number
+	workload: number
+	body: string
+	course_id: string
+	created: string
+}
+
+const ClassID: NextPage = () => {
+	const router = useRouter()
+	const [loading, setLoading] = useState<boolean>()
+	const [reviews, setReviews] = useState<[Review]>()
+	useEffect(() => {
+		setLoading(true)
+		if (router.isReady) {
+			fetch(
+				`https://omshub-readonly.gigalixirapp.com/reviews?id=${router.query.classid}`
```

Comment:
Per legal discussion, we should redact this data for now, it's still not clear what the legality is of using the OMSCentral data. Let's either `TODO` this block, or use mock data instead.